internal/lib/sandbox/namespaces.go

```diff
@@ -90,6 +91,29 @@ func (s *Sandbox) CreateNamespacesWithFunc(managedNamespaces []string, pinnsPath
 	return nsTypeToPath, nil
 }
 
+// NamespacePaths returns all the paths of the
+// namespaces of the sandbox. If a namespace is not
+// managed by the sandbox, the namespace of the infra
+// container will be returned.
+// It returns a map of nsType -> path, allowing for
+// callers to branch on the namespace type
+func (s *Sandbox) NamespacePaths() map[string]string {
```

Comment:
I think returning a structure for the namespace tuple would make it more error prone compared to a map, WDYT?

Reply:
hm yeah I agree. fixed